workers-docs/src/content/tooling/wrangler/commands.md

```diff
@@ -266,3 +263,11 @@ If you would like to be able to publish your code to multiple places, please see
 ### secret
 
   Interact with your secrets. [Check out the docs.](/tooling/wrangler/secrets)
+
+### subdomain
+
+  Create or change your [workers.dev](https://workers.dev) subdomain.
+
+  ```bash
+  wrangler subdomain <name>
+  ```
```

Comment:
Would be good to add a couple lines about what will happen to your existing Workers. 

If you've already selected a workers.dev subdomain, running `wrangler subdomain <name>` will update _all_ your current running Workers to run on the new subdomain (e.g. `hello.world.workers.dev` will now run on `hello-new.world.workers.dev`).